docs/tutorials/08_quantum_kernel_trainer.ipynb

```diff
@@ -102,7 +98,7 @@
ogSya2sxoyKE/i+sMf3dO3xYoBR6/RmtVkbpzcyonmlKqbOVUjlKqUFKqcVmDx6aPx+yeDEXZSFyWWj+fOStWsnvHhlyNNQjDQ3a8ouxTi3KUurkqX3LaHSNlRE+RpwKOyV73FTfT7xQgNnRT0EOJUTzYKHssfTd0z7fJbakIcj5ny4cnbJBVVUBdXXdN0YXZSEix7QUlfK7R3E5WvBHZs+Gmjmz26IsauZMLspC5LB14RDap03ngkhkyNGroHPqVO0kt98OaWiAqqpC5IEHMH3qVEzX9/HTDV7J3tySTEjEjukOMim0YeY9RPdJ9oa3TNNXPtSjCDjQhik9FkSafmzPqe9eT6Hc4xDp3ZEbyj2e8BpNlM+c3sNfHP/575w69dQPABG5px5FqA8DsQsixSv0AUBGzEJO3GdZWGcSz9t9qXYE2sFKDTIqXg0n1ekOnM6PZGteRq2aINfcvKrB2tlZ7tT9J5zqw/84Hz8RUcCw4CciChjPQz2xEnXe+amjLlEHopUO33hhA7OdmcmeN1l2hjaS+UzdDg+a/fycCvnY+R7tvCYY0klfrPETEQUMC34i6lPT+gUoKCjo9de0Prid++nOV6GeKLdDOsmEVGLZGfYJAqc/Xz+FBL2Uaj5Er9uCggKI9H5eKfOTtwV5BJgfscZPRBQwLPiJiALGl6Eer9gxWiTRNAqJbvBKZn1eP09RkK5hnUwKySX7XqKvO7nZnlk7yT9Y4yciCpiMqvFbqdk43dlkR4dvqnP3JzpHomkH/NyScEIm1PKNPie1fYHhKlo9l1SMp+XIccOO3JYjx/t8HTt0/SujCn4i6i1ysgBK9d4uUtDHpGynDZyY/j+I1B1DPUREAeNIjX9Lwx7XZnBMteMKME6jndMCJJrd044lDp0WtLBPJjh9LbnbOcsQj/+xxk9EFDCM8bts/I3XYsqcWcjPy4MKhxHJzUWnUQCWfCHVjlEiP3K84E+HRTsSpTHVKR2ixt94LWbMuxt5M28G6uogNTVQi5cg+3Nnnyr8jcI+XLbOOYlCa6l2jLohUeitryUVo1JdQIjXZXphjd9FU+bM0gr9DRu0DRs2IG/mzej67/9GZ45fihHKNH0vqUhBlPYFv50ToDnd4Vt+wTCgrq77xro6SFER7gp9odc5jGphqS6daMdY9Uxa5D3TuHkvC2v56Yuduy5q2fEhUFPTfWNNjbadiMglLPhdtG7eArQvXgJMmACEQsCECWhfvATr5rHmRETucSTUc3HVEGzc6H5h5vewT/3qFwEAU365EOUXDEPLjg+xbs5jp7Ynm8Zkm9ypNtWDMLbfTMeoV5LJ80SfebzvkJVrJdF9K+S9tI/xp5v61S8mXdCT+9gxSpmIoR4iooDJ2Bq/naNYzI7zB+yd3iF6XDtDWByJoXFzJs5Ur4lUORUK9Pp9UfJY4yciCpiMrfETWcGpGShIWPBbYGaUg13TO8QeI9GICD9M6ZDuI3zSYWoGq5y6FhjiSX+mQj0icpWI7BSRXSJyr9OJIiIi5ySs8YtINoAFAK4EsA9AvYi8oJTa7nTirHK6YzSWV7Vsp95POjB6v07n/aFDQHl57/nsW44c92RlqkStKT/V8s20/Hp+ppGdeyyfh6wzE+q5BMAupdRHACAiqwFcB8B3BT+R3crLAZHe243WoCVKF2ZCPZUA9sY83qdvIyKiNCQqwSIgIvJPAK5SSn1bf/wNAF9SSt3eY79bANyiP7wQwAf2J9cRAwA0e50Ik5hWxwweA5zVq24/dizw3nu999a2Gz3jinTKW6tpPUcpdaZTiSGNmVBPI4DBMY8H6du6UUo9DeBpABCRd5VS42xJocOYVmekU1qBaHr3pEV60ylv0ymtQWIm1FMPYJiIfF5EcgHcCOAFZ5NFREROSVjjV0pFROR2AK8CyAawRCm1zfGUERGRI0zdwKWUehnAyxaO+3RyyfEE0+qMdEorkF7pZVopJQk7d4mIKLNwkjYiooCxteBPp6kdRGSJiBwUEd8POxWRwSKyXkS2i8g2Efm+12mKR0TyReTPIrJVT+tDXqcpERHJFpHNIvKS12npi4jsFpH3RWSLiLzrdXoSEZF+IvJbEWkQkR0i8g9ep4k0toV69Kkd/oqYqR0ATPPj1A4AICJfBhAGsEwpdaHX6emLiJwN4Gyl1CYRKQHwHoApfsxbEREARUqpsIjkAKgD8H2l1NseJy0uEbkbwDgAZyilrvE6PfGIyG4A45RSaTGGX0R+A+AtpdQz+ojAQqXUEY+TRbC3xn9qagel1EkA0akdfEkp9SaAFq/TYYZSar9SapP+/88A7IBP755WmrD+MEf/821HkogMAvA1AM94nZZMIiKlAL4MYDEAKKVOstD3DzsLfk7t4AIRORfAaADveJyUuPTQyRYABwH8QSnl27QC+AWA2QC6PE6HGQrAayLynn6nvJ99HsCnAJbqYbRnRKTI60SRhp27aUREigE8D+BOpdRRr9MTj1KqUyl1MbS7vC8REV+G0kTkGgAHlVJeTb1gVY1SagyAqwHM0sOVfhUCMAbAQqXUaABtAHzd7xckdhb8pqZ2oOTo8fLnATyrlFrjdXrM0Jv26wFc5XFS4qkGMFmPna8GcIWIrPA2SfEppRr1fw8CWAstvOpX+wDsi2nt/RbaDwH5gJ0FP6d2cIjeYboYwA6l1GNep6cvInKmiPTT/18ArbO/wdNExaGUuk8pNUgpdS606/V/lFIzPE6WIREp0jv2oYdM/hE+nghRKdUEYK+IDNc3fQWcyt03bFt6Md2mdhCRVQAmABggIvsAPKCUWuxtquKqBvANAO/rsXMAuF+/o9pvzgbwG32UVxaAWqWUr4dJpokKAGu1OgBCAFYqpV7xNkkJ/RuAZ/WK4EcAbvI4PaTjnbtERAHDzl0iooBhwU9EFDAs+ImIAoYFPxFRwLDgJyIKGBb8REQBw4KfiChgWPATEQXM/wduCM4hFo8AewAAAABJRU5ErkJggg==\n",
```

Comment:
This text below still refers to `QuantumeKernel`

![image](https://user-images.githubusercontent.com/40241007/195886697-92f6ffd6-dcc0-42c4-8524-e5471e86c578.png)

Reply:
Replaced with "we will use an instance of `TrainableFidelityQuantumKernel`"